packages/devtools_app/lib/src/shared/release_notes/release_notes.dart

```diff
@@ -91,10 +89,26 @@ class _ReleaseNotesViewerState extends State<ReleaseNotesViewer>
       child: Stack(
         children: [
           if (child != null) child,
-          ReleaseNotes(
-            releaseNotesController: widget.releaseNotesController,
-            visibilityAnimation: visibilityAnimation,
-            markdownData: markdownData,
+          LayoutBuilder(
+            builder: (context, constraints) {
+              final widthForSmallScreen =
+                  constraints.maxWidth - 2 * densePadding;
+              final width = min(
+                _ReleaseNotesViewerState.maxViewerWidth,
+                widthForSmallScreen,
+              );
+
+              return Stack(
```

Comment:
I don't think this needs to be in a stack

Reply:
@kenzieschmoll I couldn't get this to work without a stack. It would bug out and just show a weird hazy overlay.
Do you know if I would have to do anything special to get this to work without a Stack?